src/docs/sdk/event-payloads/properties/status.mdx

```diff
@@ -1,7 +1,6 @@
 `status`
 
-: _Optional_. Describing the `status` of the Span. We use this property to determine if the [Transaction](/sdk/event-payloads/properties/transaction/)
-ended in an error or not.
+: _Optional_. Describes the `status` of the Span/Transaction.
```

Comment:
Had to remove the link because the property now doesn't exist. This appears on both Spans and Transactions and is not only about "errors", so I've trimmed the description.

Reply:
Can we add a note similar to https://github.com/getsentry/relay/blob/569678659bac656d6c4a6edcbdf3381677b5eaa2/relay-common/src/constants.rs#L153?

Reply:
Oh wait merged 😄 we can get to this later then.